examples/jsm/exporters/USDZExporter.js

```diff
@@ -426,13 +575,31 @@ ${ array.join( '' ) }
 function buildTexture( texture ) {
 
 	return `
-    def Shader "Texture_${ texture.id }"
-    {
-        uniform token info:id = "UsdUVTexture"
-        asset inputs:file = @textures/Texture_${ texture.id }.jpg@
-        token inputs:wrapS = "repeat"
-        token inputs:wrapT = "repeat"
-        float outputs:r
+	
```

Comment:
This method remains just for reference. I quickly found all the limitations for new features when having all textures in a single separate scope.